webpack/react_app/components/TargetingHosts/HostStatus/HostStatus.js

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment } from 'react';
+import { Icon } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+const HostStatus = props => {
+  const { status } = props;
+
+  switch (status) {
+    case 'cancelled':
+      return (
+        <Fragment>
```

Comment:
Fragment is only needed when there is more than one child element this and the next ones can be removed :)